bin/setup.js

```diff
@@ -47,7 +47,7 @@ if (saveRequested) {
       // - https://nodejs.org/api/path.html#path_path_parse_path
       var base = path.parse(pathToAppRoot).base
       var readMeContents = '# ' + base + '\n' + 'Created with [hoodie](https://github.com/hoodiehq)'
-      fs.writeFile(fd, readMeContents,
+      fs.writeFile(path.join(pathToAppRoot, 'README.md'), readMeContents,
```

Comment:
can we store `path.join(pathToAppRoot, 'README.md')` in a variable like `readmePath` and us it in both line 34 and 50?